### PR TITLE
Gdb 5286 introduce angular translate core

### PR DESCRIPTION
**Pull request by sava-savov-ontotext**

### CODE REVIEW

File: src/js/angular/core/directives/queryeditor/query-editor.controller.js [hunk 1]
```diff
@@ -511,19 +511,19 @@ function QueryEditorCtrl($scope, $timeout, toastr, $repositories, $modal, ModalS
...
                 $scope.toggleSampleQueries();
                 if (!edit) {
-                    toastr.success('Saved query: ' + savedQueryName + ' was deleted.');
+                    toastr.success($translate.instant('query-editor.delete.saved.query.success.msg', {savedQueryName: savedQueryName}));
                 }
             })
             .error(function (data) {
                 const msg = getError(data);
-                toastr.error(msg, 'Error! Cannot delete saved query');
+                toastr.error(msg, $translate.instant('Error! Cannot delete saved query'));
```
Comment (DesislavaBorisova) [inline src/js/angular/core/directives/queryeditor/query-editor.controller.js:519]: Missing key here

File: src/js/angular/core/directives/queryeditor/templates/query-editor.html [hunk 2]
```diff
@@ -103,25 +103,25 @@
...
                         </button>
                         <button class="btn btn-link" id="sameAs"
                                 ng-hide="notoolbarSameAs"
                                 ng-click="currentQuery.sameAs = !currentQuery.sameAs"
-                                tooltip="Expand results over owl:sameAs: {{currentQuery.sameAs ? 'ON' : 'OFF'}}"
+                                tooltip="{{createTooltipMsg('core.expand.sameAs', currentQuery.sameAs)}}"
                                 tooltip-append-to-body="true">
                             <span class="icon-2-5x"
-                                  tooltip="{{!currentQuery.inference ? 'Requires \'Include Inferred\'!' : ''}}"
+                                  tooltip="{{!currentQuery.inference ? 'core.require.inferred' : '' | translate}}"
```
Comment (DesislavaBorisova) [inline src/js/angular/core/directives/queryeditor/templates/query-editor.html:124]: Is it not meant to be `"{{!currentQuery.inference ? {{'core.require.inferred' | translate}} : ' '}}"`?

Comment (sava-savov-ontotext) [reply src/js/angular/core/directives/queryeditor/templates/query-editor.html:124]: No this is not supported

**Review by DesislavaBorisova — APPROVED** [review]
Looks good to me!

**Pull request merged**